Marlin/src/config/examples/delta/FLSUN/auto_calibrate/Configuration.h

```diff
@@ -532,19 +532,13 @@
   #if ENABLED(DELTA_AUTO_CALIBRATION)
     // set the default number of probe points : n*n (1 -> 7)
     #define DELTA_CALIBRATION_DEFAULT_POINTS 4
-
-    // Enable and set these values based on results of 'G33 A'
-    //#define H_FACTOR 1.01
-    //#define R_FACTOR 2.61
-    //#define A_FACTOR 0.87
-
   #endif
 
   #if ENABLED(DELTA_AUTO_CALIBRATION) || ENABLED(DELTA_CALIBRATION_MENU)
-    // Set the radius for the calibration probe points - max 0.9 * DELTA_PRINTABLE_RADIUS for non-eccentric probes
+    // Set the radius for the calibration probe points - max DELTA_PRINTABLE_RADIUS for non-eccentric probes
     #define DELTA_CALIBRATION_RADIUS 73.5 // mm
     // Set the steprate for papertest probing
-    #define PROBE_MANUALLY_STEP 0.025
+    #define PROBE_MANUALLY_STEP (MIN_STEPS_PER_SEGMENT / DEFAULT_XYZ_STEPS_PER_UNIT)
```

Comment:
Since `MIN_STEPS_PER_SEGMENT` is allowed to be 0, this could break manual probing.

Reply:
Some users have been known to set this to 0 so that all moves are accepted, even though 1 would also work. Since most CPUs can do a compare to zero faster than a compare to 1, setting it to 0 rather than 1 provides a tiny optimization.

Reply:
Some users have been known to set this to 0 so that all moves are accepted, even though 1 would also work.

Reply:
Also, watch out! Since this is integer math, `MIN_STEPS_PER_SEGMENT / DEFAULT_XYZ_STEPS_PER_UNIT` is likely to evaluate as 0 most of the time in any case!

Reply:
// Moves (or segments) with fewer steps than this will be joined with the next move
#define MIN_STEPS_PER_SEGMENT 5

Fewer steps than 0? I just checked : testing in planner is done with < ; so < 0 would implicate a move with a negative number of segments???

The old const int dropsegments could be set to 0 = drop zero segments.

Reply:
"Since this is integer math..." are you sure?

On my printer this is   5 / (200*16/double(2)/double(16))  and calculates correctly as 0.05 mm

Reply:
But to make sure we could change this to:

    #define PROBE_MANUALLY_STEP (MIN_STEPS_PER_SEGMENT / double(DEFAULT_XYZ_STEPS_PER_UNIT))

Reply:
> "Since this is integer math..." are you sure?

Yes. That is, if anyone ever sets `DEFAULT_XYZ_STEPS_PER_UNIT` to an integer it will be end up evaluating as integer maths.

If a desirable step size is 0.05, maybe it should just be set to a literal 0.05. One problem is that if someone doubles their micro-steps, the move is suddenly only half as big.

The point of `MIN_STEPS_PER_SEGMENT` is only to set a lower bound on a valid move size. It really isn't meant to be used this way. It would be better to simply set a fixed value here, one that is useful for manual probing and one that doesn't end up requiring many spins of the knob to move 1mm. Then, just sanity-check (`static_assert`) the value of `PROBE_MANUALLY_STEP` to make sure it isn't any smaller than `MIN_STEPS_PER_SEGMENT / float(DEFAULT_XYZ_STEPS_PER_UNIT))`.

As it is getting late there, you can go ahead and make a PR related to this tomorrow.

Reply:
"Then, just sanity-check (static_assert) the value of PROBE_MANUALLY_STEP..." :+1: